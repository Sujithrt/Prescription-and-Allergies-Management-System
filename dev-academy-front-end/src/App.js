import './App.css';
import PnamsApp from './PnamsApp';


function App() {
  return (
      <PnamsApp/>
  );
}

export default App;
