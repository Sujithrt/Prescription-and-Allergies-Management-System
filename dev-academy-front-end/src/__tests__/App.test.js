import React from 'react';
import App from '../App';
import { shallowToJson } from "enzyme-to-json";
import { shallow } from 'enzyme';

describe("App Test", () => {
    it("App is rendered as expected", () => {
        const app = shallow(<App />);
        expect(shallowToJson(app)).toMatchSnapshot();
    });
});