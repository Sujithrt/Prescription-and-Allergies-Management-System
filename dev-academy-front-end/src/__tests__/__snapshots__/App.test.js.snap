// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test App is rendered as expected 1`] = `<PnamsApp />`;
